addons/rose/tests/integration/components/rose/form-test.js

```diff
@@ -46,7 +46,10 @@ module('Integration | Component | rose/form', function (hooks) {
         as |form|
       >
         <form.input @label="Input" @name="input-field" />
-        <form.textarea @label="Textarea" @name="textarea-field" />
+        <Hds::Form::Textarea::Field @value="This is my description" name="textarea-field" as |F|>
```

Comment:
This should include disabled={{form.disabled}}, same as the below select field to show the usage